MobileFilter/sections/specific_web.txt

```diff
@@ -405,6 +405,13 @@ m.comic.sportsseoul.com##div[style*="width:100%"][style*="height:100px"]
 winfuture.mobi##iframe[src^="//j.wfcdn.de/j/iframes/outbrain_"]
 setfilmizle.vip##.menu_alti_reklam_mobil
 m.imdb.com##.cornerstone_slot
+itest.5ch.net###instant_ad
+itest.5ch.net###js-bottom-ad-300x250
+itest.5ch.net###js-bottom-ad-320x250
+itest.5ch.net###js-cardview_ad
+itest.5ch.net##.js-cardview_ad-320x50
+itest.5ch.net##.js-overlay_ad
+itest.5ch.net##.res_ad
```

Comment:
It seems that we need to add them. But what about some leftovers on the main page?

Reply:
Will fix them too.